src/javascripts/components/cookie-functions.js

```diff
@@ -161,11 +161,8 @@ export function resetCookies () {
     if (!options[cookieType]) {
       for (var cookie in COOKIE_CATEGORIES) {
         if (COOKIE_CATEGORIES[cookie] === cookieType) {
+          // Delete cookie
           Cookie(cookie, null)
```

Comment:
Any reason we can't just use `deleteCookie` here?

```suggestion
          deleteCookie(cookie)
```

Reply:
@lfdebrux Technically, no, but we do (near the top of this file) suggest using the `Cookie()` helper function for getting, setting and deleting. I guess you could say that's more for files where you're importing cookie functions though...

Reply:
Yeah, I guess although we don't have to use the external API, it is best to be consistent. It does make me wonder whether those functions should then be encapsulated, to prevent usage, but that's for another day.